lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -284,7 +284,7 @@ defmodule Mix.Tasks.Release.Init do
     if not defined RELEASE_DISTRIBUTION (set RELEASE_DISTRIBUTION=sname)
     if not defined RELEASE_BOOT_SCRIPT (set RELEASE_BOOT_SCRIPT=start)
     if not defined RELEASE_BOOT_SCRIPT_CLEAN (set RELEASE_BOOT_SCRIPT_CLEAN=start_clean)
-    set RELEASE_SYS_CONFIG=!REL_VSN_DIR!\sys
+    if not defined RELEASE_SYS_CONFIG (set RELEASE_SYS_CONFIG=!REL_VSN_DIR!\sys)
```

Comment:
NB I can't test this and I don't believe it works appropriate in dynamic `RUNTIME_CONFIG=true` scenario